NEMO/static/nemo.js

```diff
@@ -96,6 +96,78 @@ function collapse_all_categories()
 	save_sidebar_state();
 }
 
+function get_qualified_tools_for_user(user_id) {
```

Comment:
I would change the name to something more explicit since this doesn't actually "get" anything